components/cmake/modules/FindPIO.cmake

```diff
@@ -0,0 +1,24 @@
+# - Try to find PIO
+#
+# Placeholder code until spio gets a proper config.cmake
+#
+# Once done, this will define:
+#
+#   The "spio" target
+#
+
+if (NOT PIO_LIBDIR)
```

Comment:
Does this mean that the user can pass `-DPIO_LIBDIR=/path/to/my/scorpio` to use an existing installation? If so, maybe we should use a more cmake-y name, like `PIO_DIR` or `PIO_ROOT`?

Reply:
This will go away once scorpio has a good cmake config.